doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -47,15 +47,16 @@ In the current version of {PRODUCT}, an HA authoring environment is supported wi
 
 * If a {CENTRAL} pod crashes during creation of an asset, the asset might be created but not indexed, so it cannot be used. The user can open the asset in {CENTRAL} and save it again to make it indexed.
 
-You can also deploy additional managed or immutable {KIE_SERVERS}, if required. If you want to use additional {KIE_SERVERS}, you must complete an additional manual step to enable the `OpenShiftStartupStrategy` setting, as described in <<startupstrategy-enable-proc>>. With this setting, {CENTRAL} can automatically discover any {KIE_SERVERS} in the same namespace, including immutable {KIE_SERVERS} and managed {KIE_SERVERS}. This feature also requires the `OpenShiftStartupStrategy` setting on the {KIE_SERVER}. 
+* When a user deploys a service to the {KIE_SERVER}, the {KIE_SERVER} deployment is rolled out again. No user is able to deploy another service to the same {KIE_SERVER} until the roll-out completes. 
```

Comment:
Suggested rewrite:

Users can not deploy another service to the same {KIE_SERVER} until the roll-out completes.